.github/dependabot.yml

```diff
@@ -18,6 +18,23 @@ updates:
         patterns:
           - "github.com*"
 
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/pkg/apis" # Location of package manifests
```

Comment:
This seems like it will result in a separate PR, but we need this update in the same PR as the the root go.mod. But it doesn't look like we can have multiple directories in the list.

Reply:
I didn't find a better option yet. Let's go ahead and merge this and confirm if it might actually work for us.